.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
-  - "node"
+  - "7.6.0"
```

Comment:
How do we feel about following hapi to test 8, 10, and latest?

https://github.com/hapijs/hapi/blob/master/.travis.yml

Reply:
I think these versions are still very popular.

Reply:
Saw that. Was wondering too. It'd probably make more sense. I'll copy them.

Reply:
(Both v7 and v9 are already EOL: https://github.com/nodejs/Release)

Reply:
Ah well! That would be why then, haha!